SpellData.lua

```diff
@@ -83,12 +83,12 @@ local DR_KNOCKBACK = NS.diminishingReturnCategory.DR_KNOCKBACK;
 -- Events (and units) to track
 NS.trackType = {
     -- For pet kicks
-    TRACK_PET = 0, -- SPELL_CAST_SUCCESS & pet GUID
-    TRACK_PET_AURA = 1, -- SPELL_AURA_APPLIED & pet GUID, e.g., pet kicks
+    TRACK_PET = 1,
+    TRACK_PET_AURA = 2, -- SPELL_AURA_APPLIED & pet GUID, e.g., pet kicks

```

Comment:
Avoid using AURA_ events for CC abilities, that way it's hard to see if you for instance fleshcrafted/shadowmelded a CC

Reply:
Only use it for chastise (to distinguish stun or incap) and Shambling rush (to distinguish from leap)